server/website/website/tasks/periodic_tasks.py

```diff
@@ -25,7 +25,7 @@
 def run_background_tasks():
```

Comment:
Same with this file. I realize your editor is fixing the spacing but I'd revert this because it clutters the PR. I am currently fixing the formatting & style of all files now so once I get that PR merged in it should fix this problem.